pkg/util/validation/validation.go

```diff
@@ -77,13 +80,32 @@ func ValidateOverrideSpec(overrideSpec *policyv1alpha1.OverrideSpec) error {
 	if overrideSpec.OverrideRules == nil {
 		return nil
 	}
-
+	specPath := field.NewPath("spec")
```

Comment:
How about omitting the `specPath` variable?  like:
```
rulePath := field.NewPath("spec").Child("overrideRules").Index(overrideRuleIndex)
```

Reply:
I think there may be other fields of `spec` in the future that will also use the `specPath` variable.

Reply:
We usually put the variable close to the section referring to it. It'd be helpful for readability.

> I think there may be other fields of spec in the future that will also use the specPath variable.

Then, we still can use `field.NewPath("spec").Child("`the future field`")` pattern :)